template/src/components/app.js

```diff
@@ -1,32 +1,21 @@
-import { h, Component } from 'preact';
+import { h } from 'preact';
 import { Router } from 'preact-router';
 
 import Header from './header';
 
-// Code-splitting is automated for routes
+// Code-splitting is automated for `routes` directory
 import Home from '../routes/home';
 import Profile from '../routes/profile';
 
-export default class App extends Component {
-	
-	/** Gets fired when the route changes.
-	 *	@param {Object} event		"change" event from [preact-router](http://git.io/preact-router)
-	 *	@param {string} event.url	The newly routed URL
-	 */
-	handleRoute = e => {
-		this.currentUrl = e.url;
-	};
```

Comment:
I'm not entirely sure why this was added to begin with, as it was not being used in any particular way, so it was removed. I can add it back if it does serve some purpose.